PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
+def perilEndorsement():
+    print "Getting authorization token."
+
+    #URL = "https://api.develop.socotra.com/account/authenticate"
+    URL = "http://127.0.0.1:8080/account/authenticate"
+    #URL = "https://api.develop.socotra.com/account/v1/authenticateInternal
+    payload = {
+                "username": "alice.lee",
+                "password": "socotra",
+                "hostName": "vishnu-vishnu1.co.develop.socotra.com"
+                }
+
+    r = requests.post(URL, json=payload)
+    data = json.loads(r.text)
+    authorizationToken = data["authorizationToken"]
+    print("Retrieved the authorization token.")
+
+    policyResponseURL = "https://api.develop.socotra.com/policy/100005202/"
+    headers = { "content-type":"application/json" }
+    headers['Authorization'] = authorizationToken
+
+    r = requests.get(policyResponseURL, headers=headers)
+    data = json.loads(r.text)
+    exposures = data['exposures']
+    exposureLocator = exposures[0]['locator']
+
+    exposure = exposures[0]
+    perils = exposure['perils']
+
+    peril = perils[1]
+    perilLocatorToRemove = peril['locator'].encode('ascii', 'ignore')
+    perilLocatorToUpdate = perils[2]['locator'].encode('ascii', 'ignore')
+
+    deductible = "100"
+    lumpSumPayment = "200"
+
+    perilUpdateRequest = dict([("addFieldGroups", []),
+                               ("fieldValues", {}),
+                               ("perilLocator", perilLocatorToUpdate),
+                               ("removeFieldGroup", []),
+                               ("updateFieldGroups", []),
+                               ("deductible", deductible),
+                               ("lumpSumPayment", lumpSumPayment),
+                               ("removeIndemnityPerEvent", False),
+                               ("removeIndemnityPerItem", False)])
```

Comment:
I think we've already established that you know that you don't need to cast a list of tuples to create a dictionary:
```
    perilUpdateRequest = {'addFieldGroups': [],
                          'fieldValues': {},
                          'perilLocator': perilLocatorToUpdate,
                          'removeFieldGroup': [],
                          'updateFieldGroups': [],
                          'deductible': deductible,
                          'lumpSumPayment': lumpSumPayment,
                          'removeIndemnityPerEvent': False,
                          'removeIndemnityPerItem': False}
```